.python-version

```diff
@@ -1 +1 @@
-3.13
+3.12
```

Comment:
Let's not backdate Python here, local dev isn't really impacted by `pylint` not working on 3.13.1

Reply:
Is there any value in developing in 3.13 if we can't get past CI?

Reply:
Sure you make a good point, we can have local dev be 3.13